keel-ec2-plugin/src/main/kotlin/com/netflix/spinnaker/keel/ec2/resource/ClusterHandler.kt

```diff
@@ -262,11 +294,46 @@ class ClusterHandler(
   private fun ResourceDiff<ServerGroup>.isCapacityOnly(): Boolean =
     current != null && affectedRootPropertyTypes.all { it == Capacity::class.java }
 
+  /**
+   * @return `true` if the only changes in the diff are to scaling.
+   */
+  private fun ResourceDiff<ServerGroup>.isAutoScalingOnly(): Boolean =
+    current != null &&
+      affectedRootPropertyTypes.any { it == Scaling::class.java } &&
```

Comment:
Isn't this caught by the clause below?

Reply:
No?

Reply:
Hmm, ok. I read that as "all affected properties are either `Capacity` or `Scaling`", whereas this line read "any affected property is `Scaling`", which seemed redundant.